src/test/ui/block-result/issue-3563.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0599]: no method named `b` found for type `&Self` in the current scope
+error[E0599]: no method named `b` found for reference `&Self` in the current scope
```

Comment:
The only thing I have reservations about is this `reference` here, but that should probably be changed in `prefix_string` to skip references or change it to be something like `borrowed enum`.